SeaPublicWebsite/appsettings.json

```diff
@@ -52,8 +52,4 @@
         "CookieName": "_ga",
         "BaseUrl": "https://www.google-analytics.com"
     },
-    
-    "GlobalConstants": {
-        "ServiceName": "Improve your property’s energy efficiency"
-    }

```

Comment:
This was not used anywhere (I think I created it before creating the `Constants` class